tools/start_all_tmux.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+# This script is used to start all of the microservices
+# for the Lucida project by creating a tmux window for each
+# server in the background.
+# Example calls:
+# $./start_all_tmux.sh secure
+# $./start_all_tmux.sh
+# To attach to the tmux session
+# use the following command:
+# $tmux a -t lucida
+
+SESSION_NAME="lucida"
```

Comment:
where's the check to see if you're already in a tmux session? What if I'm in a tmux session that is not called lucida?